frontend/src/components/molecules/MatchItem/MatchItem.tsx

```diff
@@ -87,6 +88,31 @@ export const MatchItem = (props: MatchItemProps,) => {
    const isMatchFinished = Boolean(props.matchStatus === MatchStatus.FINISHED);
    const isMatchStarted = Boolean(props.matchStatus === MatchStatus.STARTED);
 
+   const agreeDialogCallback = (matchId: string, teamName: string): void => {
```

Comment:
Jeśli teamName to teamId (przynajmniej tak jest użyty poniżej), to bym od razu to tak zmieniła.

Reply:
Dzięki! TeamId potrzebuję, a tą nazwę muszę zmienić!